manimo/actuators/arms/franka_arm.py

```diff
@@ -171,14 +198,22 @@ def _default_policy(self, action_space, kq_ratio=1.5, kqd_ratio=1.5):
                 robot_model=self.robot.robot_model,
                 ignore_gravity=True,
             )
+        elif action_space == ActionSpace.OSC:
+            return OperationalSpaceLowFreq(
+                q_current=q_initial,
+                call_freq=self.hz,
+                interm_freq=self.hz*4,
+                Kp=kq,
+                Kd=kqd,
+            )
 
     def _get_desired_pos_quat(self, eef_pose):
         if self.delta:
             ee_pos_cur, ee_quat_cur = self.robot.get_ee_pose()
             ee_pos_desired = ee_pos_cur + torch.Tensor(eef_pose[:3])
 
             # add two quaternions
-            ee_quat_desired = torch.Tensor(quat_add(ee_quat_cur, eef_pose[3:]))
+            ee_quat_desired = torch.Tensor(quat_add(eef_pose[3:], ee_quat_cur))
```

Comment:
double checking, is it intentional to change the order here, this doesn't seem correct

Reply:
not sure about the ordering. this change is probably from the original repo. I'll revert this then